tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature

```diff
@@ -1,3 +1,5 @@
+# this scenario is skipped on ocis because it opens share folder which in not possible in OCIS
+# but it works for OC10 see issue https://github.com/owncloud/web/issues/6896 for more detail
```

Comment:
mentioned comment at the top of the feature  film because this feature file is related to `accept/decline shares coming from internal users` that means somehow this comment will be connected